src/itemlistformaction.cpp

```diff
@@ -482,56 +482,61 @@ bool ItemListFormAction::process_operation(Operation op,
 		}
 		break;
 	case OP_MARKFEEDREAD:
-		LOG(Level::INFO, "ItemListFormAction: marking feed read");
-		try {
-			const auto message_lifetime = v->get_statusline().show_message_until_finished(
-					_("Marking feed read..."));
-
-			std::vector<std::string> guids;
-			for (const auto& item : visible_items) {
-				const std::string guid = item.first->guid();
-				guids.push_back(guid);
-			}
-			rsscache->mark_items_read_by_guid(guids);
-
-			if (apply_filter) {
-				// We're only viewing a subset of items, so mark them off one by one.
-				v->get_ctrl()->mark_all_read(guids);
-			} else {
-				v->get_ctrl()->mark_all_read(pos);
-			}
-
-			if (visible_items.size() > 0) {
-				std::lock_guard<std::mutex> lock(feed->item_mutex);
-				bool notify = visible_items[0].first->feedurl() != feed->rssurl();
+		if (!cfg->get_configvalue_as_bool(
+				"confirm-mark-feed-read") ||
+			v->confirm(_("Do you really want to mark this feed as read (y:Yes n:No)? "),
+				_("yn")) == *_("y")) {
+			LOG(Level::INFO, "ItemListFormAction: marking feed read");
+			try {
+				const auto message_lifetime = v->get_statusline().show_message_until_finished(
+						_("Marking feed read..."));
+	
```

Comment:
Apparently this and other empty lines contain a tab; please remove those tabs.

Reply:
I believe this has been fixed.

Reply:
Oh, never mind I just realized what you were talking about. I will actually resolve it shortly.

Reply:
Doesn't look like it: https://github.com/newsboat/newsboat/pull/1782/checks?check_run_id=3655285773

Reply:
Should finally be fixed, or at the very least the automatic format checker is not yelling at me anymore.